ui/src/app/service/group/group.service.ts

```diff
@@ -31,6 +31,14 @@ export class GroupService {
         return this._http.get<Group[]>('/group');
     }
 
+    /**
+     * Get all groups that the user can access, without the default group.
+     * @returns {Observable<Group[]>}
+     */
+    getGroupsWithoutDefault(): Observable<Group[]> {
```

Comment:
why not pass an optional parameters to `getGroups(withoutDefault?: boolean)` and then use http query parameters of angular